packages/manager/modules/exchange/src/exchange.template.js

```diff
@@ -395,6 +394,18 @@ export default /* @ngInject */ ($templateCache) => {
     'exchange/account/outlook/activate/account-outlook-activate.html',
     accountOutlookActivateTemplate,
   );
+  $templateCache.put(
```

Comment:
😅

Reply:
Yes, need a large refactoring for this module.